packages/components/src/panel/body.native.js

```diff
@@ -15,9 +19,10 @@ export class PanelBody extends Component {
 	}
 
 	render() {
-		const { children } = this.props;
+		const { children, title } = this.props;
 		return (
-			<View >
+			<View style={ styles.section }>
+				{ title && <Text style={ styles.sectionHeader }>{ title.toUpperCase() }</Text> }
```

Comment:
uppercasing programmatically can cause trouble in different languages according to what locale is used underneath, so this doesn't look like an ideal solution to me. Is there a recommended way for doing this @gziolo ?

Reply:
We can handle it from css and add `text-transform: uppercase` property to section title.

Reply:
I guess, it's a design decision and it should be always handled outside of JS code. In the case of web, this is also an accessibility concern. Uppercase is read differently by assistive technologies. I have no idea how it translates to native. I would personally render this title as is and apply style modifications as suggested.

Reply:
If we will follow the `Option B: Fully Neutral` for Thomas style guidelines we will omit that issue.